res/themes/light/css/_light.scss

```diff
@@ -198,9 +198,8 @@ $event-highlight-bg-color: $yellow-background;
 $event-timestamp-color: #acacac;
 
 $copy-button-url: "$(res)/img/feather-customised/clipboard.svg";
-$collapse-button-url: "$(res)/img/feather-customised/minimise.svg";
-$expand-button-url: "$(res)/img/feather-customised/maximise.svg";
```

Comment:
Hmm, to be honest, I don't really see the purpose of a variable for the image file name like this... Why not just use the path directly? Also, I dislike how it affects the naming: most users call it maximise, but then reference the variable called expand.

Perhaps let's just remove these two path variables.